build/BuildExtension.ps1

```diff
@@ -14,11 +14,19 @@
  # limitations under the License.
  ##>
 [CmdletBinding()]
-Param([string[]]$TasksToBuild, [switch]$SkipInit, [switch]$SkipCompile, [switch]$SkipTest)
+Param(
+    [string[]]$TasksToBuild,
+    [switch]$SkipInit,
+    [switch]$SkipCompile,
+    [switch]$SkipTest,
+    [string]$Publisher,
+    [string]$Version
+)
 
 
 pushd (Join-Path $MyInvocation.MyCommand.Path ..)
-$functionsModule = Import-Module ./BuildFunctions.psm1 -PassThru
+$functionsModule = Import-Module ./BuildFunctions.psm1 -PassThru -Verbose:$false
```

Comment:
Should this be configurable instead of hard coding in false?

(same on line 70)

Reply:
`-Verbose` here just lists the imported functions. It is unlikely to be useful, and the file is easy enough to edit locally for the rare occasion when it is.